packages/react/src/components/pickers/BasePicker.tsx

```diff
@@ -350,6 +365,7 @@ export class BasePicker<T extends {}, P extends IBasePickerProps<T>>
             )}
           </div>
         </SelectionZone>
+        {this.renderError(classNames.error)}
```

Comment:
This would need to be associated with the input through `aria-describedby` when it is available, without overriding or replacing the other `aria-describedby` value. The `aria-describedby` attribute can take a space-separated list of ids that are concatenated in order as the description.

Textfield does this by inserting the error message in the same element as the static description, so sharing the same parent `id` association.